packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -96,20 +96,31 @@ pub struct ExperimentConfig {
     )]
     pub num_workers: usize,
 
-    /// Allows a JavaScript runner to grow to some initial size in megabytes before triggering
-    /// garbage collections.
+    /// Heap size in megabytes of the V8 runtime in each JavaScript runner under which garbage
+    /// collection doesn't occur.
+    /// See "--num-workers" to set the number of JavaScript runners
+    /// executing in parallel.
     ///
-    /// This is useful when it is known that experiments need a certain minimum heap to run to
-    /// avoid repeatedly invoking the garbage collector when growing the heap.
-    /// When used in the wrong conditions this could waste memory.
+    /// This setting is most of the time better off left to its default value.
+    /// It could be beneficial if your are going to store a large amount of "eternal" (=lives as
+    /// long as the simulation is running) data on the heap.
+    /// Example: you know that at any given time you have 500MB of unreachable data on the
+    /// heap. You could set this argument to 600MB (500MB plus some) and save some runs of the
+    /// garbage collector.
+    ///
+    /// Defaults to V8's default.
```

Comment:
V8's default for what? Can we give the arg this corresponds to

Reply:
Sorry I perhaps misused the term arg. It's a V8 flag, `max-old-space-size` you can see it referred to in Node and Deno.

Reply:
Added the V8 flag https://github.com/hashintel/hash/pull/446/commits/d8ee2e580c6fca35b0e97341db9b2874dd3e8a4a